docs/design/operator.md

```diff
@@ -349,8 +349,9 @@ spec:
     subvolumeGroup: csi
     kernelMountOptions: readdir_max_bytes=1048576,norbytes
     fuseMountOptions: debug
+    radosNamespace: rados-test-cephfs
   rbd:
-    radosNamespace: rados-test
+    radosNamespace: rados-test-rbd
```

Comment:
There is not equivalence between the RBD RADOS NS and the RADOS NS for CephFS metadata.
With that in mind, please rename this back to: 
```suggestion
    radosNamespace: rados-test
```